src/ai-dialog-footer.ts

```diff
@@ -1,44 +1,46 @@
+import {inject} from 'aurelia-dependency-injection';
```

Comment:
Also one note is you can put it right beside the constructor which improves legibility and refactoring.